### ISSUE TITLE
Doesn't support android API 22, version 5.1

### ISSUE THREAD

**Issue by northern-fox** [fullstackreact/react-native-oauth#26]
My code works for Android API 23, version 6.0 emulator. And doesn't work for ex. android API 22, version 5.1
I get the following
![screenshot from 2016-12-16 15 10 17](https://cloud.githubusercontent.com/assets/8543255/21260950/fe5c012e-c3a1-11e6-8efc-5bfb772fd014.png)

Could you please add in your documentation which API level is supports your lib.

**Comment by pewh** [comment]
+1 same issue

**Comment by paradisensei** [comment]
+1 same issue

**Comment by auser** [comment]
Yes, I believe that's the webview issue. I'll have to set up some time to test the different android versions.
